day-to-day.md

```diff
@@ -179,14 +170,13 @@ Every team member is welcome to choose their hardware, software, and development
 
 - Slack for daily communication
 - Github for remote versioning control
-- Github Projects for project management
+- Jira for project management
 - Google Suite for document storage and email.
 
 Team members recommend some of the following tools for personal workflow:
```

Comment:
I think I'd be a proponent of just getting rid of this whole section and limiting the list to tools that we all have to use for the company to operate successfully as a business (i.e. Gsuite, Slack, Zoom, Harvest, Jira, Github, etc) and keeping personal/staff recommended apps to slack or some place more on-demand/current.